java/res/xml-sw600dp/key_period.xml

```diff
@@ -42,6 +42,16 @@
                 latin:keySpec="\?"
                 latin:moreKeys="!text/morekeys_tablet_period,!text/morekeys_question" />
         </case>
+        <case
+            latin:keyboardLayoutSet="bengali_avro"
```

Comment:
Since bengali dahri is same as devagari danda, this can be written above in line 28

```
           latin:languageCode="hi|bn"
           latin:keyboardLayoutSet="hindi|qwerty"
```

This will work without even creating a new layout for avro.